templates/skeleton/app/routes/cart.$lines.tsx

```diff
@@ -0,0 +1,69 @@
+import {redirect, type LoaderArgs} from '@shopify/remix-oxygen';
+
+/**
+ * Automatically creates a new cart based on the URL and redirects straight to checkout.
+ * Expected URL structure:
+ * ```ts
+ * /cart/<variant_id>:<quantity>
+ *
+ * ```
+ * More than one `<variant_id>:<quantity>` separated by a comma, can be supplied in the URL, for
+ * carts with more than one product variant.
+ *
+ * @param `?discount` an optional discount code to apply to the cart
```

Comment:
I know this comes from demo-store but I think `@param` is not a good fit for showing a query-string. Not sure what's the best way but maybe just mention it in `@example`?

When we compile TS to JSDoc, the `@param` tags need to be used for function parameters.